src/compiler/crystal/command.cr

```diff
@@ -102,7 +102,11 @@ class Crystal::Command
     when File.file?(command)
       run_command(single_file: true)
     else
-      error "unknown command: #{command}"
+      if command.ends_with?(".cr")
+        error "file not found: '#{command}'"
```

Comment:
Can we use the same error message as below? (File '...' does not exist). Also, I think all error messages here shouldn't be capitalized (so the other one must be changed).